inference-engine/src/transformations/CMakeLists.txt

```diff
@@ -23,15 +23,12 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 
 add_library(${TARGET_NAME} SHARED ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 
-target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_API)
+target_compile_definitions(${TARGET_NAME} PRIVATE inference_engine_transformations_EXPORTS)
```

Comment:
we don't have to define it. It's automatically defined as `<shared library name>_EXPORTS`

Reply:
Got it. Fixed.